apps/api/src/app/events/usecases/digest-filter-steps/digest-filter-steps.usecase.ts

```diff
@@ -46,4 +46,19 @@ export class DigestFilterSteps {
       _templateId: command.templateId,
     };
   }
+  public static getNestedValue<ObjectType>(payload: ObjectType, path: string) {
```

Comment:
Please add a blank line to separate the methods.

Would you mind to add a unit test suite located in this same folder as `digest-filter-steps.usecase.spec.ts` where we can test this function?

Reply:
Actually this suggested code has small bug, this line result = payload[key]; should be result = result[key];